doc/contributing/api-documentation.md

```diff
@@ -1,11 +1,13 @@
 # Node.js API Documentation Tooling
 
 The Node.js API documentation is generated by an in-house tooling that resides
-within the [tools/doc](https://github.com/nodejs/node/tree/main/tools/doc)
-directory.
+in the [nodejs/api-docs-tooling](https://github.com/nodejs/api-docs-tooling)
+repository.
+
+It is accessible in this repository from the `tools/doc` directory, where it is installed as an npm package.
 
 The build process (using `make doc` or `make doc-only`) uses this tooling to
-parse the markdown files in [`doc/api/`][] and generate the following:
+parse the markdown files in \[`doc/api/`]\[] and generate the following:
```

Comment:
```suggestion
parse the markdown files in [`doc/api/`][] and generate the following:
```

Reply:
@flakey5 this still needs to be resolved. You re added the link, but you still need to unescape the above